dev/prod/webpack.config.js

```diff
@@ -152,6 +152,7 @@ module.exports = {
   ],
   devtool: prod ? false : 'source-map',
   devServer: {
+    hot: true,
```

Comment:
Hot reloading works without this stuff, do we need it?

Reply:
Removed.